rna/rna-collect-multiple-metrics/README.md

```diff
@@ -0,0 +1,28 @@
+# RNACollectMultipleMetrics with Picard
+
+This workflow runs `picard CollectInsertSizeMetrics`, `picard CollectAlignmentSummaryMetrics`, `picard QualityScoreDistribution`, and `picard MeanQualityByCycle` on a BAM file
```

Comment:
This workflow runs the following `picard` commands on a BAM file:

- [CollectInsertSizeMetrics](https://software.broadinstitute.org/gatk/documentation/tooldocs/4.0.1.2/picard_analysis_CollectInsertSizeMetrics.php)
- [CollectAlignmentSummaryMetrics](https://software.broadinstitute.org/gatk/documentation/tooldocs/4.0.1.2/picard_analysis_CollectAlignmentSummaryMetrics.php)
- [QualityScoreDistribution](https://software.broadinstitute.org/gatk/documentation/tooldocs/4.0.1.2/picard_analysis_QualityScoreDistribution.php)
- [MeanQualityByCycle](https://software.broadinstitute.org/gatk/documentation/tooldocs/4.0.1.2/picard_analysis_MeanQualityByCycle.php)